spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -63,6 +63,98 @@
         before do
           request.headers['Authorization'] = socure_secret_key
         end
+
+        shared_examples 'repeats webhooks' do
+          let(:headers) do
+            {
+              Authorization: socure_secret_key_queue.last,
+              'Content-Type': 'application/json',
+            }
+          end
+          let(:endpoints) do
+            (1..3).map { |i| "https://#{i}.example.test/endpoint" }
+          end
+          let(:repeated_body) do
+            b = {}.merge(webhook_body)
+            b[:event].delete(:data)
+            b
+          end
+          before do
+            allow(IdentityConfig.store)
+              .to receive(:socure_docv_webhook_repeat_endpoints).and_return(endpoints)
+
+            allow(DocumentCaptureSession).to receive(:find_by)
+              .and_return(create(:document_capture_session))
```

Comment:
is it possible to create the sessions with known IDs ahead of time so we don't have to stub the find?